site/netlify.toml

```diff
@@ -1,3 +1,23 @@
 [build]
+  base = "site/"
+  command = "hugo"
+  publish = "../publishedSite"
+
 [build.environment]
-HUGO_VERSION = "0.88.1"
```

Comment:
Of the two config files, I suspect this one is being used, because the build showed hugo version 0.88.1.